pkgs/development/python-modules/flask-gravatar/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, flask
+, pytest-runner
```

```suggestion
, flask
```
Please remove unused inputs and pytest-runner is useless in nixpkgs.